frontend/src/lib/components/Tooltip.tsx

```diff
@@ -37,7 +37,11 @@ export function Tooltip({
     const child = React.isValidElement(children) ? children : <span>{children}</span>
 
     return props.title ? (
-        <AntdTooltip {...props} visible={isDefaultTooltip ? visible : localVisible && debouncedLocalVisible}>
+        <AntdTooltip
+            {...props}
+            visible={isDefaultTooltip ? visible : localVisible && debouncedLocalVisible}
+            color={'#2D2D2D'}
```

Comment:
Pretty sure we shouldn't hardcode colors into JSX, but do something cleaner... 🤔

Reply:
It's a tooltip prop: https://ant.design/components/tooltip/

Any recommendations for making it cleaner?

And also, are we okay with making this global change? (cc: @clarkus )

Reply:
into global.scss

```css
.ant-tooltip { 
   background: var(--something-predefined);
}
```

Reply:
Got it, thanks!